src/components/SideBar.tsx

```diff
@@ -75,22 +80,19 @@ class SideBar extends React.Component<SideBarGlobalProps, XCSpyState> {
         );
     }
 
-    getComponentList(): Anchor[] {
+    getComponentList() {
         const list = [];
         const props = this.props;
         const components = props.components;
         components.map((component: string) => {
             list.push(
-                <Anchor
-                    primary={true}
+                <Link
                     key={component}
                     value={component}
-                    className={(props.currentComponent === component) ? "active" : ""}
-                    onClick={(e) => {
-                        props.setCurrentComponent(e.target.getAttribute("value"));
-                    }}>
+                    to={{ pathname: "/app/", search: `${SERVER_URL}=${props.serverUrl}&${API}=${props.api}&${CURRENT_COMPONENT}=${component}` }}
```

Comment:
you should put the pathname in a constant.
We don't want to get hard coded paths everywhere in our code